addons/hr/static/src/scss/hr.scss

```
@@ -58,3 +58,9 @@
 .o_form_label.o_hr_form_label {
     cursor: default;
 }
+
+.hr_presence {
+    font-size: 0.75em;
+    vertical-align: middle;
+  }
+
```

Wrong indentation and one extra too many :innocent: 
```suggestion
}
```